src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -1,8 +1,28 @@
+// Node modules.
 import React from 'react';
 import moment from 'moment';
-
+// Relative imports.
 import * as customPropTypes from '../prop-types';
 
+const onDownloadLink = (form) => (event) => {
+  // Derive if the download is a PDF.
+  const isPDF = form?.attributes?.url.toLowerCase().includes('.pdf');
```

Comment:
Can we abstract this logic into a helper function since `SearchResult` is also using this? Or, inside `SearchResult`, we can just _not_  do an `onClick` function if `isPDF` is `false`. That way we know `isPDF` is true if this function is called.

Reply:
Done! 🎉